cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/brokers/DatacenterBrokerHeuristicExample.java

```diff
@@ -121,65 +138,130 @@ public DatacenterBrokerHeuristicExample() {
         Log.setLevel(Level.WARN);
 
         System.out.println("Starting " + getClass().getSimpleName());
-        this.vmList = new ArrayList<>();
-        this.cloudletList = new ArrayList<>();
 
-        simulation = new CloudSim();
+        // false: for the best fit version in the CloudletToVmMappingBestFit example
+        // true: for the best fit broker
+        boolean useBestFitBroker = true;
+        seed = 0;
+
+        // Heuristic
+        random0 = new UniformDistr(0, 1, seed);
+
+        simulation0 = new CloudSim();
+
+        final Datacenter datacenter0 = createDatacenter(simulation0);
+
+        broker0 = createBroker();
 
-        final Datacenter datacenter0 = createDatacenter();
+        vmList0 = createVms(random0);
+        cloudletList0 = createCloudlets(random0);
+        broker0.submitVmList(vmList0);
+        broker0.submitCloudletList(cloudletList0);
 
-        DatacenterBrokerHeuristic broker0 = createBroker();
+        simulation0.start();
 
-        createAndSubmitVms(broker0);
-        createAndSubmitCloudlets(broker0);
+        // BestFit
+        random1 = new UniformDistr(0, 1, seed);
 
-        simulation.start();
+        simulation1 = new CloudSim();
 
-        final List<Cloudlet> finishedCloudlets = broker0.getCloudletFinishedList();
-        new CloudletsTableBuilder(finishedCloudlets).build();
+        final Datacenter datacenter1 = createDatacenter(simulation1);
+
+        if (useBestFitBroker) {
+            broker1 = new DatacenterBrokerBestFit(simulation1);
+        }
+        else {
+            broker1 = new DatacenterBrokerSimple(simulation1);
+            broker1.setVmMapper(this::bestFitCloudletToVmMapper);
```

Comment:
You best-fit implementation is correct, so we should simply remove this `bestFitCloudletToVmMapper` and only use the class you provided. The same method should be removed from `CloudletToVmMappingBestFit` and the example update to your class.

Providing 2 best-fit implementations will just confuse users.